build-parent/pom.xml

```diff
@@ -584,6 +595,99 @@
             <artifactId>consul-client</artifactId>
             <version>${version.com.orbitz.consul}</version>
           </dependency>
+
+          <!-- MicroProfile -->
+          <dependency>
+            <groupId>org.eclipse.microprofile.config</groupId>
+            <artifactId>microprofile-config-api</artifactId>
+            <version>${version.microprofile-config}</version>
+          </dependency>
+
+          <dependency>
+            <groupId>org.eclipse.microprofile.config</groupId>
+            <artifactId>microprofile-config-tck</artifactId>
+            <version>${version.microprofile-config}</version>
+            <scope>test</scope>
+          </dependency>
+
+          <dependency>
+            <groupId>org.wildfly</groupId>
+            <artifactId>wildfly-microprofile-config-implementation</artifactId>
+            <version>${version.wildfly-microprofile-config}</version>
+          </dependency>
+
+          <dependency>
+            <groupId>org.wildfly.swarm</groupId>
+            <artifactId>microprofile-config-api</artifactId>
+            <version>${version.wildfly-microprofile-config}</version>
+          </dependency>
+
+          <dependency>
+            <groupId>org.eclipse.microprofile.fault-tolerance</groupId>
+            <artifactId>microprofile-fault-tolerance-api</artifactId>
+            <version>${version.microprofile-fault-tolerance}</version>
+          </dependency>
+
+          <dependency>
+            <groupId>org.eclipse.microprofile.fault-tolerance</groupId>
+            <artifactId>microprofile-fault-tolerance-tck</artifactId>
+            <version>${version.microprofile-fault-tolerance}</version>
+            <scope>test</scope>
+          </dependency>
+
+          <dependency>
+            <groupId>org.eclipse.microprofile.health</groupId>
+            <artifactId>microprofile-health-api</artifactId>
+            <version>${version.microprofile-health}</version>
+          </dependency>
+
+          <dependency>
+            <groupId>org.eclipse.microprofile.health</groupId>
+            <artifactId>microprofile-health-tck</artifactId>
+            <version>${version.microprofile-health}</version>
+            <scope>test</scope>
+          </dependency>
+
+          <dependency>
+            <groupId>org.eclipse.microprofile.metrics</groupId>
+            <artifactId>microprofile-metrics-api</artifactId>
+            <version>${version.microprofile-metrics}</version>
+          </dependency>
+
+          <dependency>
```

Comment:
Should the TCK dependencies be added to testsuite/microprofile-tck instead?

I presume they're not required for building?

Reply:
No, they're not. I will move them.